lib/hooks/useFeatureFlags.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { Flags } from "@lib/cache/flags";
+import { createContext, useContext, useState } from "react";
+
+const featureFlagsContext = createContext({
```

Comment:
To keep this consistent we should use `FeatureFlagsContext` here

Reply:
Nice, thanks - completely missed that. I guess because it's not a component React didn't complain about the casing.